compat.h

```diff
@@ -169,6 +170,14 @@ void	warnx(const char *, ...);
 #define O_DIRECTORY 0
 #endif
 
+#ifndef FNM_CASEFOLD
+#ifdef FNM_IGNORECASE
+#define FNM_CASEFOLD FNM_IGNORECASE
+#else
+#define FNM_CASEFOLD 0x10
```

Comment:
I'm not too keen on this but noticed that `libtar` does something similar (see [libtar:compat/compat.h](https://github.com/tklauser/libtar/blob/6379b5d2ae777dad576aeae70566740670057821/compat/compat.h#L46)).